it/controllers/studentLoans/PglAmountControllerISpec.scala

```diff
@@ -145,366 +56,297 @@ class PglAmountControllerISpec extends IntegrationTest with ViewHelpers with Emp
-          lazy val result = {
-            dropEmploymentDB()
-            authoriseAgentOrIndividual(scenarioData.isAgent)
-            insertCyaData(EmploymentUserData(
-                          sessionId,
-                          mtditid,
-                          nino,
-                          taxYearEOY,
-                          employmentId, isPriorSubmission = true, hasPriorBenefits = false, hasPriorStudentLoans = true,
-                          EmploymentCYAModel(
-                            EmploymentDetails(
-                              employerName = "Whiterun Guards",
-                              employerRef = Some("223/AB12399"),
-                              startDate = Some("2022-04-01"),
-                              didYouLeaveQuestion = Some(false),
-                              taxablePayToDate = Some(3000.00),
-                              totalTaxToDate = Some(300.00),
-                              currentDataIsHmrcHeld = false
-                            )
-                          )))
-            userDataStub(IncomeTaxUserData(), nino, taxYearEOY)
-
-            urlGet(url(taxYearEOY), follow = false, welsh = scenarioData.isWelsh,
-              headers = Seq(HeaderNames.COOKIE -> playSessionCookies(taxYearEOY)))
-          }
-
-          result.status shouldBe SEE_OTHER
-          result.headers("Location").headOption shouldBe Some(controllers.studentLoans.routes.StudentLoansCYAController.show(taxYearEOY, employmentId).url)
-        }
-
-        "redirect to student loans cya page when there is no employment user data returned" in {
-          lazy val result = {
-            dropEmploymentDB()
-            authoriseAgentOrIndividual(scenarioData.isAgent)
+    "render the postgraduate amount page when there is no prior or cya data" in {
+      lazy val result = {
+        dropEmploymentDB()
+        authoriseAgentOrIndividual(false)
+        insertCyaData(EmploymentUserData(
+          sessionId,
+          mtditid,
+          nino,
+          taxYearEOY,
+          employmentId, isPriorSubmission = false, hasPriorBenefits = false, hasPriorStudentLoans = false,
+          EmploymentCYAModel(
+            EmploymentDetails(
+              employerName = "Whiterun Guards",
+              employerRef = Some("223/AB12399"),
+              startDate = Some("2022-04-01"),
+              didYouLeaveQuestion = Some(false),
+              taxablePayToDate = Some(3000.00),
+              totalTaxToDate = Some(300.00),
+              currentDataIsHmrcHeld = false
+            ),
+            studentLoans = Some(StudentLoansCYAModel(
+              uglDeduction = false, uglDeductionAmount = None, pglDeduction = true, pglDeductionAmount = None))
+          )))
+        userDataStub(IncomeTaxUserData(), nino, taxYearEOY)
+
+        urlGet(url(taxYearEOY), false, headers = Seq(HeaderNames.COOKIE -> playSessionCookies(taxYearEOY)))
```

Comment:
The welsh content parameter boolean is defaulted to 'false' and so can be removed.

Reply:
Done for all of the occurrences